src/commands/logs.js

```diff
@@ -5,24 +5,27 @@ const LogV2 = require('../models/log.js');
 const Log = require('../models/log-v4.js');
 const Logger = require('../logger.js');
 const { Deferred } = require('../models/utils.js');
+const colors = require('colors/safe');
 
 async function appLogs (params) {
-  const { alias, after: since, before: until, search, 'deployment-id': deploymentId } = params.options;
+  const { alias, addon, after: since, before: until, search, 'deployment-id': deploymentId } = params.options;
```

Comment:
suggestion: we could alias it here to `addonId` and reuse it directly on line 22